lib/tests/test_working_copy.rs

```diff
@@ -210,6 +211,14 @@ fn test_checkout_file_transitions(use_git: bool) {
     wc.check_out(repo.op_id().clone(), None, &right_tree)
         .unwrap();
 
+    // jujutsu doesn't create submodules, so we manually create the
+    // directory here
+    for (_left_kind, right_kind, path) in &files {
+        if matches!(*right_kind, Kind::GitSubmodule) {
```

Comment:
Can this use `==` instead of `matches!`?

Reply:
Added Deriving `PartialEq`, `Eq` to `Kind` and switched to `==`